packages/rich-text/src/plugins/Hyperlink/ToolbarIcon.js

```diff
@@ -27,7 +27,7 @@ export default class HyperlinkToolbarIcon extends Component {
         type={INLINES.HYPERLINK}
         icon="Link"
         title="Hyperlink"
-        onToggle={event => this.handleClick(event)}
+        onToggle={(event) => this.handleClick(event)}
```

Comment:
Can we just pass the function by reference here? i.e.
```jsx
onToggle={this.handleEvent}
```
since it's already instance-bound?

Reply:
:+1: I knew I missed it in one of those files.